components/emoji_picker/components/emoji_picker_item.jsx

```diff
@@ -75,6 +75,7 @@ export default class EmojiPickerItem extends React.Component {
             image = (
                 <img
                     alt={'emoji image'}
+                    data-testid={emoji.aliases}
```

Comment:
I think this should have a test id, but I don't think you are using it in the test code anywhere currently.

Reply:
I use this in order to create the array of emojis's names.